### PR TITLE
Lets admins pick frequency/volume in playsound global, saves me audacity time

### PR DESCRIPTION
**Pull request by silicons**
Honk!

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
also stop playing all star midis

File: code/modules/admin/verbs/playsound.dm [hunk 1]
```diff
@@ -7,7 +7,10 @@
 	log_admin("[key_name(src)] played sound [S]")
 	message_admins("[key_name_admin(src)] played sound [S]")
 
-	var/freq = 1
+	var/freq = input(usr,"What frequency? Default : 1. 2 is twice as fast, -1 is backwards. Anything above or below 100/-100 respectively will be treated as Hz instead of multiplier.", "Frequnecy", 1) as null|num
```
Comment (Thunder12345) [inline code/modules/admin/verbs/playsound.dm]: Frequnecy

File: code/modules/admin/verbs/playsound.dm [hunk 2]
```diff
@@ -27,12 +30,15 @@
 
 	SSblackbox.add_details("admin_verb","Play Global Sound") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
-
 /client/proc/play_local_sound(S as sound)
 	set category = "Fun"
 	set name = "Play Local Sound"
 	if(!check_rights(R_SOUNDS))
 		return
+	var/freq = input(usr,"What frequency? Default : 1. 2 is twice as fast, -1 is backwards. Anything above or below 100/-100 respectively will be treated as Hz instead of multiplier.", "Frequnecy", 1) as null|num
```
Comment (Thunder12345) [inline code/modules/admin/verbs/playsound.dm]: Frequnecy

**Review by Thunder12345 — COMMENTED** [review]
Would it be better to replace these two procs with one Play Sound proc, with an extra input for range, with 0 range playing globally?

File: code/modules/admin/verbs/playsound.dm (hunk 1 above)
Comment (kevinz000) [inline code/modules/admin/verbs/playsound.dm]: It's frequency

Comment (Militaires) [inline code/modules/admin/verbs/playsound.dm]: fregguency

**Comment by Militaires** [comment]
if you're gonna be scientific and name pitch "frequency" then go all out and name volume "amplitude"

**Comment by PKPenguin321** [comment]
I've wanted to just make a DJ panel for a while now but >tgui

**Comment by PKPenguin321** [comment]
also 👎 lowering volume in audacity takes like 0.1 seconds and also lowers filesize, and im not sure why youd need frequency changes on demand like this aside from awful sounds abuse, and this makes even more steps to playing sounds

**Comment by Haevacht** [comment]
Just make your shitty midispam RIGHT you nerd.

**Comment by optimumtact** [comment]
Now it's fucking three popups before one sound

stop

**Comment by kevinz000** [comment]
*shrug*

**Pull request closed without merging**